addons/sourcemod/scripting/include/API.inc

```diff
@@ -89,5 +89,8 @@ public int API_GetMaxClients()
 
 public int API_GetUserFlagBits(int client)
 {
-	return GetUserFlagBits(client);
-}
+	if (!IsClientConnected(client))
+		return -1;
```

Comment:
i think its 0 when he has no flags

Reply:
You are right. `Returns client access flags. If the client is not an admin, the result is always 0.`